### DISCUSSION TITLE
when uploading new binary file versions such as .jpg or png then the old one will not be replaced

### DISCUSSION THREAD

**Discussion by dsyleixa** [community/community#60717] [Programming Help]
### Select Topic Area

Question

### Body

when uploading new binary file versions such as .jpg or png then the old one will not be replaced.
e.g., I have a subfolder
https://github.com/dsyleixa/RaspberryPi/tree/master/qt/GameProgramming%20Qt/shootergame-tut8
containing e.g.,
https://github.com/dsyleixa/RaspberryPi/blob/master/qt/GameProgramming%20Qt/shootergame-tut8/screenshot_tut8.jpeg
https://github.com/dsyleixa/RaspberryPi/blob/master/qt/GameProgramming%20Qt/shootergame-tut8/player.png

Every time when I upload new versions then the old ones in this subfolder do not become replaced/overwritten, neither when I upload these single files in this subfolder, nor when I upload the entire related folder in the parent folder with all the files.

That is absolutely annoying, tbh.

**Comment by ossanna16** [comment]
Hi @dsyleixa ,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Raspberry Pis. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by dsyleixa** [comment]
Thanks, but actually this it a github issue, it is not specifically related to RaspberryPi.
As mentioned, it is about the fault that when uploading new version of  image files then they do not become updated by this action at github.

**Comment by ManosPerpirakes** [comment]
personally, I delete the old files from the repository (i.e. the ones to replace) and then upload the new ones

**Comment by dsyleixa** [comment]
I also did that arbitrarily, optionally, but then the same fault as before.
Even when deleting the entire subdir and uploading the entire updated subdir anew.
Probably a webbsite buffer mess.

**Comment by ManosPerpirakes** [comment]
I had the same issue with .png files recently and I noticed that they need to be renamed in order to be replaced (for example if the file is "new picture.png" it could be renamed as "new_picture.png")

**Reply by dsyleixa** [reply]
but that is absolutely weird, tbh....
